client/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const POPOVER_CONTENT_CLASS =
+  "flex flex-col ml-4 max-h-[320px] w-full min-w-[320px]";
```

Comment:
Wouldn't be easier just updating the classes of the PopoverContent in its file and allowing override them when needed?

Reply:
`POPOVER_CONTENT_CLASS ` is only used for the sidebar popups, there are other instances in the app that don't use this. I will change the name to make it more clear.